### ISSUE TITLE
Use a salt?

### ISSUE THREAD

**Issue by fourkbomb** [ncss-2015-group-4/trivia#4]
The lack of security is real. You might also want to sanitise your DB inputs.

**Comment by nhardy** [comment]
For a guide on how to properly hash passwords, see the password_hash() function at https://github.com/ncss-2014-group-3/word-by-word/blob/master/database/user.py

**Comment by auscompgeek** [comment]
I don't even want to know who decided to MD5 the passwords...

**Comment by auscompgeek** [comment]
Oh, and database inputs were already sanitised from day 0, unless we did something horribly wrong.

**Comment by paked** [comment]
I think we should just use bcrypt like the rest of the sane world?

http://www.mindrot.org/projects/py-bcrypt/

**Comment by fourkbomb** [comment]
:+1: :shipit:

**Comment by paked** [comment]
Fixed with af9823ab038b0918f0c31142a8fe364a804ba286

**Comment by auscompgeek** [comment]
tl;dr: `pbkdf2_hmac`
